### DISCUSSION TITLE
Can Podman command be used in WSL2 in the same way as Docker?

### DISCUSSION THREAD

**Discussion by tonycody** [containers/podman#15880] [Q&A] [answered]
# Docker Desktop
1.The WSL2 Windows subsystem includes an Ubuntu
2.Install Docker Desktop

The Docker command is available either in Powershell or in the WSL2 subsystem Ubuntu or other WSL2 subsystems.

In PowerShell
![image](https://user-images.githubusercontent.com/6773121/191396386-4588e5d8-d4d2-4f38-bd51-5714c448e542.png)
In Ubuntu for WSL2
![image](https://user-images.githubusercontent.com/6773121/191397431-2d9479de-4793-4673-94b3-57d223491fd3.png)
In Debian for WSL2
![image](https://user-images.githubusercontent.com/6773121/191397443-4519c569-1fd7-4875-9825-1b25960c299c.png)

Podman can only be used in PowerShell. Is it supported?
![image](https://user-images.githubusercontent.com/6773121/191397548-3d422d3f-2f22-47f6-aff1-589be97b2c8c.png)

**Comment by TheComputerM** [comment]
Is there a wsl instance called `podman-machine-default` when you run `wsl -l -v`?

**Comment by rhatdan** [comment]
@n1hility PTAL

**Comment by n1hility** [comment, marked as answer]
Yes, you can run a podman package on your own WSL distro like your Ubuntu example, but if you want to use podman on Windows, with the ability to use it both on the command prompt and on the WSL prompt, then you need to use the Podman machine distribution that Podman for Windows installs and uses as part of the podman machine init command.
